src/Whoops/Run.php

```diff
@@ -293,6 +293,24 @@ public function handleException(Exception $exception)
     }
 
     /**
+     * This is a private implementation detail, you are not supposed to call this.
+     */
+    final public function handleExceptionFromPHP($exception)
```

Comment:
we could also do something like

```
// php 5.3 hack
$that= $this;
set_exception_handler(function() {
  $that->{self::EXCEPTION_HANDLER}();
} use ($that));
```

instead of what we currently have
```
set_exception_handler(array($this, self::EXCEPTION_HANDLER));
```

so we dont need this `handleExceptionFromPHP` method to be public